docker/snippets/app.conf

```diff
@@ -0,0 +1,13 @@
+
+location / {
+  send_timeout 10m;
+  include snippets/cors.conf;
+
+  client_max_body_size 100M;
```

Comment:
Does this limit apply to GET as well? If so we need to increase this limit because we already have solution files bigger than 100M.

Reply:
I used the configuration actually in production, what the max size you think we should allow ?

Reply:
If the limit applies to GETs then 200M would be better because we already have solution of 134M

Reply:
Ok, nice of @senhalil to put the "GET" back in your answer. This only concerns the size of the incoming requests so I won't change it.

Reply:
no worries happy to help :ok_hand: